lib/actions/click.js

```diff
@@ -49,11 +49,11 @@ async function simulateInputEvents(options) {
 }
 
 async function click(selector, options = {}, ...args) {
```

Comment:
Can we please add a unit test for this case ? https://github.com/getgauge/taiko/blob/6645c6d884560cb1f2d33008e7ead49ff7d7d269/test/unit-tests/click.test.js#L149 a case with other click option and proximity selector